suggest_engine/suggest.py

```diff
@@ -1,20 +1,44 @@
-from typing import List, Tuple, Set
-from data_class.file_data import FileData
+from typing import List, Tuple, Set, Callable
+from data_structure.file_data import FileData
+from util.remove_punctuation_util import process_sentence
 import string
 
 
 def help_get_score(user_input: str, sentence_substring: str, score: int, score_to_sub: int) -> int:
+    """
+
+    Calculate a modified score based on user input and a sentence substring.
+
+    This function takes a user's input, a sentence substring, an initial score, and a score reduction value.
+    It compares characters of the user input and sentence substring and calculates the modified score.
+
+    :param user_input: A string containing the user's input.
+    :param sentence_substring: A string containing a substring of a sentence to compare with.
+    :param score: An integer representing the initial score.
+    :param score_to_sub: An integer representing the score reduction value.
+    :return: An integer representing the modified score.
+    """
     if user_input[:5] == sentence_substring[:5]:
         return score - score_to_sub
-    for iterator in range(5):
```

Comment:
Better to use a constant value instead.
(applicable to all the places in the file the value is used)

Reply:
Done